core/src/components/cat-tooltip/cat-tooltip.scss

```diff
@@ -0,0 +1,24 @@
+:host {
+  display: contents;
+}
+
+.tooltip {
+  position: absolute;
+  padding: 5px 10px;
+  background-color: rgba(0, 0, 0, .7);
```

Comment:
needs to be added to the tokens project. I'd go with _color.ui.background.tooltip_ and _color.ui.font.tooltip_ and _opacity.tooltip_. Then use these consistently in the style sheet via `cat-token('color.ui....');`